deploy/helm_charts/dc_website/templates/deployment.yaml

```diff
@@ -75,9 +75,9 @@ spec:
             periodSeconds: 10
           resources:
             limits:
-              memory: "8G"
+              memory: "3G"
```

Comment:
For my own understanding, why are we lowering memory limits, both here and in line 204&206?

Reply:
We have two way of generating these yamls. Kustomize and helm (different tools). We deploy to autopush/staging/prod/stanford using kustomize, and custom DC uses helm. 

Since the [kustomize template](https://github.com/datacommonsorg/website/blob/master/deploy/base/deployment.yaml) uses 3G and 1G respectively, it means that it should be enough for custom DC as well. Since we're trying to save cost for custom DC users, we only have 1 machine for the entire GKE. I would like to conserve resources in case we need to deploy more things.

Reply:
That makes sense, the flask server won't use too much memory, only the svg mixer needs high memory.